backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ComponentDatabaseHandler.java

```diff
@@ -194,9 +194,9 @@ private void autosetReleaseClearingState(Release releaseAfter, Release releaseBe
 
     public List<Component> getMyComponents(String user) {
         //This call could be sped up, because we want the full documents
```

Comment:
Shouldn't this comment be removed, now that you've switched to loading full components in the next line?